client/my-sites/site-settings/credentials-context.jsx

```diff
@@ -0,0 +1,257 @@
+	},
+
+	renderHeader: function() {
+		const {
+			connectionTypeText,
+			connectionTypeDescription,
+			isActiveConnection
+		} = this.props;
+
+		return (
+			<table className="credentials-context">
+				<tbody>
+					<tr>
+						<td className={ isActiveConnection ? 'status-icon green' : 'status-icon gray' }>
+							{ isActiveConnection
+								? ( <Gridicon icon="checkmark-circle" /> )
+								: ( <Gridicon icon="add-outline" /> )
+							}
+						</td>
+						<td>
+							<h2>{ connectionTypeText }</h2>
+							<h3>{ connectionTypeDescription }</h3>
+						</td>
+					</tr>
+				</tbody>
+			</table>
+		);
+	},
+
+	renderPublicKey: function() {
+		return (
+			<FormTextarea onChange={ this.handleChangeEvent } name="publicKey" />
+		);
+	},
+
+	render: function() {
+		const {
+			siteId,
+			connectionType,
+			existingCreds,
+			isPressable
+		} = this.props;
+
+		return (
+			<FoldableCard
+				clickableHeader
+				header={ this.renderHeader() }
+			>
+				{
+					isPressable
+					? ( <Notice>This site is hosted by Pressable. SFTP credentials are automatically managed by our partnership.</Notice> )
+					: null
+				}
+				<table>
+					<tbody>
+						<tr>
+							<td>
+								<FormLabel>{ translate( 'Server Address' ) }</FormLabel>
+								<FormTextInput
+									onChange={ this.handleChangeEvent }
+									name="serverAddress"
+									placeholder="yoursite.com"
+									value={ formState.getFieldValue( this.state.form, 'serverAddress' ) }
+								/>
+							</td>
+							<td>
+								<FormLabel>{ translate( 'Port Number' ) }</FormLabel>
+								<FormTextInput
+									onChange={ this.handleChangeEvent }
+									name="serverPort"
+									placeholder="22"
+									value={ formState.getFieldValue( this.state.form, 'serverPort' ) }
+								/>
+							</td>
+						</tr>
+						<tr>
+							<td colSpan="2">
+								<FormLabel>{ translate( 'Username' ) }</FormLabel>
+								<FormTextInput
+									onChange={ this.handleChangeEvent }
+									name="username"
+									placeholder="username"
+									value={ formState.getFieldValue( this.state.form, 'username' ) }
+								/>
+							</td>
+						</tr>
+						<tr>
+							<td colSpan="2">
+								<FormLabel>{ translate( 'Password' ) }</FormLabel>
+								<FormPasswordInput
+									onChange={ this.handleChangeEvent }
+									name="password"
+									placeholder="password"
+									value={ formState.getFieldValue( this.state.form, 'password' ) }
+								/>
+							</td>
+						</tr>
+						<tr>
+							<td colSpan="2">
+								<FormLabel>{ translate( 'Upload Path' ) }</FormLabel>
+								<FormTextInput
+									onChange={ this.handleChangeEvent }
+									name="uploadPath"
+									placeholder="/var/www/yoursite.com/"
+									value={ formState.getFieldValue( this.state.form, 'uploadPath' ) }
+								/>
+							</td>
+						</tr>
+						<tr>
+							<td colSpan="2">
+								<FormLabel>{ translate( 'Public Key' ) }</FormLabel>
```

Comment:
Hi! I've found a possible matching string that has already been translated **74** times:
`translate( 'Public Key:' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).